cinch/roles/jenkins_master/templates/enable-kerberos-sso.groovy

```diff
@@ -0,0 +1,21 @@
+def plugin = com.sonymobile.jenkins.plugins.kerberossso.PluginImpl.getInstance()
```

Comment:
Should this file be idempotent and support check_mode?  If not, let's add a comment to explain why for future maintenance.

Reply:
We've opened issue #149 to address the kerberos-sso plugin configuration changes.